SuperBuild.cmake

```diff
@@ -314,7 +314,7 @@ Slicer_Remote_Add(SlicerDMRI
   GIT_REPOSITORY "${git_protocol}://github.com/SlicerDMRI/SlicerDMRI"
```

Comment:
If you use `SlicerDMRI/SlicerDMRI@f3a8545` in commit message it will be referenced